rest-api/dao/participant_counts_over_time_service.py

```diff
@@ -0,0 +1,156 @@
+from .participant_summary_dao import ParticipantSummaryDao
+from model.participant_summary import ParticipantSummary
+from participant_enums import EnrollmentStatus
+
+class ParticipantCountsOverTimeService(ParticipantSummaryDao):
+
+  def __init__(self):
+    super(ParticipantSummaryDao, self).__init__(ParticipantSummary)
+
+
+  def get_filtered_results(self, start_date, end_date, filters, stratification='ENROLLMENT_STATUS'):
+    """Queries DB, returns results in format consumed by front-end
+
+    :param start_date: Start date string, e.g. '2018-01-01'
+    :param end_date: End date string, e.g. '2018-01-31'
+    :param filters: Objects representing filters specified in UI
+    :param stratification: How to stratify (layer) results, as in a stacked bar chart
+    :return: Filtered, stratified results by date
+    """
+
+    start_date = start_date.replace('-', '')
+    end_date = end_date.replace('-', '')
+
+    filters_sql = self.get_facets_sql(filters)
+
+    if stratification == 'TOTAL':
+      strata = ['TOTAL']
+      sql = """
+        SELECT
+            SUM(ps_sum.cnt * (ps_sum.day <= calendar.day)) registered_count,
+            calendar.day start_date
+        FROM calendar,
+        (SELECT COUNT(*) cnt, DATE(ps.sign_up_time) day
+        FROM participant_summary ps
+        GROUP BY day) ps_sum
+        WHERE calendar.day >= :start_date
+        AND calendar.day <= :end_date
+        GROUP BY calendar.day
+        ORDER BY calendar.day;
+      """
+    elif stratification == 'ENROLLMENT_STATUS':
+      strata = [str(EnrollmentStatus(val)) for val in EnrollmentStatus]
```

Comment:
I don't think you need the EnrollmentStatus(val) here... just use str(val)?

Reply:
Ping?

Reply:
[Done](https://github.com/all-of-us/raw-data-repository/blob/cf84290ff49f52defa834a180aa0f138b7361aaa/rest-api/dao/participant_counts_over_time_service.py#L34).